docs/config.md

```diff
@@ -199,21 +199,28 @@ Obviously, you would only set one line, don't copy them all in!
 ## Editing diffs
 
 The `ui.diff-editor` setting affects the tool used for editing diffs (e.g.
-`jj split`, `jj amend -i`). The default is `meld`. The left and right
-directories to diff are passed as the first and second argument respectively.
+`jj split`, `jj amend -i`). The default is `meld`.
+
+`jj` replaces the following arguments:
+
+- `$left` and `$right` are replaced with the paths to the left and right
+  directories to diff respectively.
+
+If no interpolation parameters are found, `$left $right` is appended.
 
 For example:
 
 ```toml
-ui.diff-editor = "kdiff3" # Use merge-tools.kdiff3.edit-args
-ui.diff-editor = ["kdiff3", "--merge"] # Specify edit-args inline
+# Use merge-tools.kdiff3.edit-args
+ui.diff-editor = "kdiff3"
+# Specify edit-args inline, $left $right will be added by default
+ui.diff-editor = ["kdiff3", "--merge"]
 ```
 
-Custom arguments can be added, and will be inserted before the paths to diff:
-
 ```toml
 # merge-tools.kdiff3.program = "kdiff3"      # Defaults to the name of the tool if not specified
-merge-tools.kdiff3.edit-args = ["--merge", "--cs", "CreateBakFiles=0"]
+merge-tools.kdiff3.edit-args = [
+    "--merge", "--cs", "CreateBakFiles=0", "$left", "$right"]
```

Comment:
I wouldn't change these and the defaults in `merge-tools.toml`, but it depends on people's opinion about my comment below about continuing to rely on this behavior.